README.md

```diff
@@ -4,29 +4,232 @@ Rubiz
 [![codecov.io](https://codecov.io/github/rubicon-project/rubiz/coverage.svg?branch=master)](https://codecov.io/github/rubicon-project/rubiz?branch=master)
 [![scaladoc](https://javadoc-badge.appspot.com/com.rubiconproject/rubiz_2.11.svg?label=scaladoc)](https://javadoc-badge.appspot.com/com.rubiconproject/rubiz_2.11)
 
+## Table of Contents
+* [Purpose](#purpose)
+* [Installation](#installation)
+* [Additions](#additions)
+  * [Catchable](#catchable)
+  * [Task](#task)
+  * [Either](#either)
+  * [Try](#try)
+* [Tests](#tests)
+* [Continuous Integration](#continuous-integration)
+* [Release](#release)
 
 ## Purpose
 
 Provides much needed syntax and missing conversions for Scalaz 7.1.
 
-## Adding to sbt
+## Installation
 
 Add the following to your `build.sbt` file:
+
 ```
-libraryDependencies += "com.rubiconproject" %% "rubiz" % "0.2.0"
+libraryDependencies += "com.rubiconproject" %% "rubiz" % "0.2.+"
 ```
 
 Import all the additional syntax with `import rubiz.syntax.all._`. Specific subsets of syntax can be
 imported if preferred (eg, `import rubiz.syntax.either._`).
 
-### Running tests
+## Additions
+
+### Catchable
+
+```scala
+import scalaz.Catchable
+import scalaz.syntax.catchable._
+import scalaz.effect.IO
+import rubiz.syntax.catchable._
+```
+
+#### ensure
+Check the result inside `Catchable` to see if it matches your predicate. If it doesn't, the left becomes your provided value.
+
+```scala
+(IO("username")
+  .ensure(new Exception("Can't make a user without a name."))(_.nonEmpty)
+  .unsafePerformIO)
+// res0: String = username
+```
+
+### Task
+
+```scala
+import scalaz.concurrent.Task
+import rubiz.syntax.task._
+```
+
+#### withTiming
+Wraps timing information up with the result of the task. If the task
+failed, there isn't a result to wrap up with, and no timing information
+will be available. This is most useful for local logging of timing
+information.
+
+```scala
+(Task.now(List("Australia", "Japan"))
+  .withTiming           // Task[(FiniteDuration, List[String])]
+  .map {
+      case (timing, result) =>
+        println(s"${result.length} country names were returned by the DB in ${timing.toMillis} ms.")
+        result
+  }
+  .run)
+// 2 country names were returned by the DB in 2 ms.
```

Comment:
It's an example, a bit more useful to consider timing on things like that than a literal list. Can you think of a better way to exemplify that without an actual DB, or make it clear that the print statement is an example?